src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -10,18 +11,27 @@
 public class DriverManager {
 
 	private Job2dDriver currentDriver = new LoggerDriver();
-
+	private CompositeDriver composite = new CompositeDriver();
 	/**
 	 * @param driver Set the driver as current.
 	 */
 	public synchronized void setCurrentDriver(Job2dDriver driver) {
 		currentDriver = driver;
 	}
 
+	public synchronized void addDriver(Job2dDriver driver){
+		composite.addDriver(driver);
+		System.out.println(composite);
```

Comment:
Please, do not print to sysout. If you really need it use logger.

Reply:
It was for debugging purposes only, removed in 2a48b9a